cmd/up.go

```diff
@@ -356,7 +356,30 @@ func (up *UpContext) devMode(isRetry bool, d *appsv1.Deployment, create bool) (b
 		}
 	}
 
-	p, err := pods.GetDevPod(up.Context, up.Dev, up.Client, create)
+	reporter := make(chan string)
+	go func() {
+		long := false
+		message := ""
+		ticker := time.NewTicker(1000 * time.Millisecond)
+		for {
+			select {
+			case <-ticker.C:
```

Comment:
Do we need both the ticker and the message case? Maybe simplify it by having a single case that receives the message and then waits a second before checking again?